test/dlc_tests/sanity/test_security_check.py

```diff
@@ -105,6 +106,11 @@ def test_ecr_scan(image, ecr_client, sts_client, region):
     image_account_id = get_account_id_from_image_uri(image)
     image_repo_name, original_image_tag = get_repository_and_tag_from_image_uri(image)
     additional_image_tags = get_all_the_tags_of_an_image_from_ecr(ecr_client, image)
+    if not is_image_available_locally(image):
+        LOGGER.info(f"Image {image} not available locally!! Pulling the image...")
+        run(f"docker pull {image}")
```

Comment:
This statement needs a step before it to ensure that the docker login to the ECR registry is done and that the credentials are still valid.

```suggestion
        test.test_utils.login_to_ecr_registry(context, account_id, region)
        run(f"docker pull {image}")
```